test/Sentry.Unity.Tests/SessionIntegrationTests.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using Sentry.Unity.Tests.Stubs;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Sentry.Unity.Tests
+{
+    public class SessionIntegrationTests
+    {
+        [UnityTest]
+        public IEnumerator SessionIntegration_SomethingElse()
+        {
+            yield return null;
```

Comment:
Don't we need to load the scene first?

Reply:
There is nothing in our scenes that I need and when the test starts the active scene changes to `InitTestScene{RandomNumber}` anyway. But I might be missing something?